experimental/examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -150,7 +151,23 @@ compose.desktop {
 
 compose.experimental {
     web.application {}
-    uikit.application {}
+    uikit.application {
+        bundleIdPrefix = "org.jetbrains"
+        projectName = "FallingBalls"
+        developmentTeam = "N462MKSJ7M"
```

Comment:
Maybe remove the team?

Reply:
Yeah, it may works on simulator without developmentTeam

Reply:
done